### ISSUE TITLE
Error in AbstractPhysicalQuantityTest::getTestUnitOfMeasure

### ISSUE THREAD

**Issue by pierpaolocira** [PhpUnitsOfMeasure/php-units-of-measure#63]
Hi,
I think there is an error in the abovementioned function (related to the `MockBuilder` behaviour).

I'm trying to write a new test function, related to the following function declared into the `AbstractPhysicalQuantity` (see #62) 
```
public static function isUnitDefined($name) {
    	$units = static::getUnitDefinitions();
    	foreach ($units as $unit) {
    		if ($name === $unit->getName() || $unit->isAliasOf($name)) {
    			return true;
    		}
    	}
    	return false;
    }
```

Following the existing code, I tried to create the related unit test also adding a new unit. This is the only important part of the function to reproduce the error:
```
public function testIsUnitDefined(){
    	$newUnit = $this->getTestUnitOfMeasure('noconflict', ['definitelynoconflict']);
    	Wonkicity::addUnit($newUnit);
    	$this->assertTrue(Wonkicity::isUnitDefined("noconflict"));    	
    	$this->assertTrue(Wonkicity::isUnitDefined("definitelynoconflict"));    	
    }
```
The first assert is correct, but not the second one.

I made several test to inestigate on the error... and I saw that if you run (into `AbstractPhysicalQuantity`) a `$unit->getName()` or a `$unit->getAliases()` you will see the expected values.

So the problems seems to be linked to the `isAliasOf()` method.

I'm running on a PHP 5.5.36 environment.

Hope this helps.

**Comment by pierpaolocira** [comment]
Hi, in the aforementioned pull request (#64) I implemented an alternative method to `getTestUnitOfMeasure()` (namely `getTestUnitOfMeasureSafe()`) as a workaround for this issue in the unit tests.
This method can be removed when this issue is closed with a "real" solution (instead of my workaround).
I hope this is accepted.

**Comment by triplepoint** [comment]
I think #65 should fix this (though it's a little cumbersome).  The travis tests are running right now, if they pass I'll merge that, and you should be able to back out the workaround you made in #64.

**Comment by triplepoint** [comment]
Fixed with #65
